typings/internal-electron.d.ts

```diff
@@ -97,7 +105,7 @@ declare namespace Electron {
   interface WebPreferences {
     disablePopups?: boolean;
     embedder?: Electron.WebContents;
-    type?: 'backgroundPage' | 'window' | 'browserView' | 'remote' | 'webview' | 'offscreen';
+    type?: 'backgroundPage' | 'window' | 'remote' | 'webview' | 'offscreen';
```

Comment:
This is a breaking change which we should either find a way to make backwards compat or mention in the list of breaking changes. `WebContent.getType()` likely also needs to be updated.

I have some old code in a library of mine which checks for this type
https://github.com/samuelmaddock/electron-browser-shell/blob/c47bcb3c7a1ff274b469ecffb25245cf580e5811/packages/shell/browser/main.js#L67-L79

Reply:
Hm, yeah, I guess that's true. Updated to have WebContents that are created through `new BrowserView` return this from `getType()`.